src/Tribe/Attendee_Registration/Template.php

```diff
@@ -71,6 +71,16 @@ public function setup_context( $posts ) {
 			return $posts;
 		}
 
+		if ( wp_is_block_theme() ) {
+			error_log(
```

Comment:
Do we have an alternative for `error_log`? My reasoning behind this is so that the issue is "visible". When we were QA'ing the page would infinitely load and we had no idea why.